src/vanilla/utils/devtools.ts

```diff
@@ -67,7 +66,7 @@ export function devtools<T extends object>(
   }
 
   let isTimeTraveling = false
-  const devtools = extension.connect({ name })
+  const devtools = extension.connect({ name, ...options })
```

Comment:
Shouldn't we remove `enabled` from `options`?

Reply:
I agree, I didn't actually figure out exactly why it's there.

Reply:
`enabled` is used in our lib, right?